src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -123,3 +125,35 @@ def get(self, request: Request, organization: Organization) -> HttpResponse:
                 "chunk_ids": [el["chunk_id"] for el in chunk_ids],
             },
         )
+
+
+@region_silo_endpoint
+class OrganizationProfilingChunksFlamegraphEndpoint(OrganizationProfilingBaseEndpoint):
+    def get(self, request: Request, organization: Organization) -> HttpResponse:
+        if not features.has("organizations:profiling", organization, actor=request.user):
+            return Response(status=404)
+
+        params = self.get_snuba_params(request, organization, check_global_views=False)
+
+        project_ids = params.get("project_id")
```

Comment:
`project_id` is not passed for my projects. This should check params to see if it resolved to a single project or be a projects endpoint.

Reply:
@Zylphrex marking this as resolved as we've discussed and sorted this out this over Slack